roles/lnls-ans-role-servweb/handlers/main.yml

```diff
@@ -0,0 +1 @@
+---
```

Comment:
Maybe remove the directory `handlers` altogether if we are not using it.

Reply:
deleted